components/odh-notebook-controller/controllers/notebook_network.go

```diff
@@ -129,11 +128,12 @@ func CompareNotebookNetworkPolicies(np1 netv1.NetworkPolicy, np2 netv1.NetworkPo
 }
 
 // NewNotebookNetworkPolicy defines the desired network policy for Notebook port
-func NewNotebookNetworkPolicy(notebook *nbv1.Notebook) *netv1.NetworkPolicy {
+func NewNotebookNetworkPolicy(notebook *nbv1.Notebook, log logr.Logger, namespace string) *netv1.NetworkPolicy {
 	npProtocol := corev1.ProtocolTCP
+	log.Info("Controller is running in namespace", "namespace", namespace)
```

Comment:
This message gets printed too frequently to my taste, and the logger already has a `namespace` key (namespace of the notebook being processed, so the log message looks weird.

```
{"level":"info","ts":"2024-11-26T08:35:59Z","logger":"controllers.Notebook","msg":"Controller is running in namespace","notebook":"jupyter-nb-htpasswd-2dcluster-2dadmin-2duser","namespace":"rhods-notebooks","namespace":"redhat-ods-applications"}
```

Reply:
@shalberd thanks